### PR TITLE
196 stats error

### LINKED ISSUE #196: Stats Page Error
Says 'Internal Server Error' if you go to the stats page and you haven't played a game yet

### PR DESCRIPTION
**Pull request by Rebecca-CYLi**
fixes #196 

adds basic scores and increments as player plays the game.
tested it, increments and fetches scores just fine.

### CODE REVIEW

**Comment by HamdiaA** [comment]
I still get the error for some reason..

File: flask/app/auth.py [hunk 1]
```diff
@@ -42,10 +42,11 @@ def register():
             user = {
                 "username": username,
                 "password": generate_password_hash(password),
+                "gameScore": {"draws": 0, "wins": 0, "losses": 0},
             }
             Users.insert_one(user).inserted_id
             flash("Register successful, please login.")
-            return redirect(url_for("index"))
+            return redirect(url_for("rps.index"))
```
Comment (calvh) [inline flask/app/auth.py:49]: `auth.login`

File: README.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Next, change the following specifications in `docker-compose.yaml`:
     nginx:
         image: jonasal/nginx-certbot:dev
         volumes:
-            - ./nginx/localhost_conf.d:/etc/nginx/user_conf.d
```
Comment (calvh) [inline README.md:83]: revert

File: flask/app/auth.py (hunk 1 above)
Comment (Rebecca-CYLi) [reply flask/app/auth.py]: done

File: README.md (hunk 2 above)
Comment (Rebecca-CYLi) [reply README.md]: done, will need to relook into it in another branch

**Review by calvh — APPROVED** [review]
thanks @Rebecca-CYLi lgtm

**Pull request merged**